packages/run-protocol/src/econ-behaviors.js

```diff
@@ -143,7 +143,7 @@ export const setupAmm = async ({
 
   if (!ammPublicFacet) {
     // ISSUE: what is this test for?!
-    throw new Error(`ammPublicFacet broken  ${ammPublicFacet}`);
+    throw Error(`ammPublicFacet broken  ${ammPublicFacet}`);
```

Comment:
could be a good time to answer "what is this test for?!" seems like it's a safety assert. should use `assert` instead?

Reply:
Agree that it should use `assert`, as well as `X` (aka `assert.details`). As written, it reveals a stringification to `ammPublicFacet` to callers, which is likely ok but a bad habit to fall into, as well as denying the logging system access to the actual `ammPublicFacet` object.

Reply:
I think ti should just be removed,.

Reply:
The test in order to confirm that the primary goal of that method worked, which is that it made an amm.